src/shared/components/rightbar/RightBar.tsx

```diff
@@ -24,7 +24,8 @@ export default class RightBar extends React.Component<{}, {}> {
     };
 
     private CancerTypeDescendantStudies(cancerList:CancerType[]) {
-        return cancerList.filter(cancer => cancer.name !== "Other" && cancer.name.toLowerCase().indexOf("pancancer") < 0).map((filteredCancer:CancerType) => (
+        return cancerList.filter(cancer => (cancer.cancerTypeId !== "other" &&  cancer.cancerTypeId !== "mixed"))
```

Comment:
you don't have to do this here but more readable and flexible method for writing the predicate here would be 
['mixed','other'].includes(cancer.cancerTypeId).